### DISCUSSION TITLE
Custom Logo in Panel

### DISCUSSION THREAD

**Discussion by taufik-nurrohman** [mecha-cms/mecha#138] [Ideas]
You can replace the navicon SVG path with another SVG icon path which you can get via <https://materialdesignicons.com>

``` php
$_['lot']['bar']['lot'][0]['lot']['folder']['icon'] = 'M17,15l1.55,1.55c-0.96,1.69-3.33,3.04-5.55, …';
```

**Result:**

![2021-08-22 22:51:54](https://user-images.githubusercontent.com/1669261/130361785-e3708801-5646-4e52-a014-4435096b616b.png)

In the future, there should be some kind of way to use native image or SVG element with complex rules.

**Comment by nurbaitifitriyani** [comment]
Where is the path to changing the logo in the extension panel, Mas?

**Reply by taufik-nurrohman** [reply]
Bisa dinyatakan di mana saja di lokasi `.\lot\x\*\index\panel.php` dan `.\lot\y\*\index\panel.php`

Atau secara manual melalui kait:

~~~ php
Hook::set('_', function($_) {
    ​$​_​[​'lot'​][​'bar'​][​'lot'​][​0​][​'lot'​][​'folder'​][​'icon'​] = ​'M17,15l1.55,1.55c-0.96,1.69-3.33,3.04-5.55, …'​;
    return $_;
});
~~~
